### ISSUE TITLE
A bug in "collection" folder

### ISSUE THREAD

**Issue by V3NK47** [tomahawk-player/tomahawk#519]
Every time i open tomahawk the collection turns to zero. Every time i have to open configure tomahawk and add my folder to it, so it scans again and finds my library..its so annoying not to have collection by default every time.

**Comment by Arunva28** [comment]
Also, the collection does not update itself even after control->Updated Collections

**Comment by muesli** [comment]
Do you have "Watch for changes" enabled and your network share not mounted on Tomahawk start up, maybe? Also, which version of Tomahawk?

**Comment by Arunva28** [comment]
Yes, Watch for changes is enabled and i am currently using v0.8

**Comment by muesli** [comment]
Do you also access your music from a network share? Which is not mounted/accessible when starting Tomahawk?

In other words, is Tomahawk starting up, not finding the music on the share and then deciding it's wiping the collection?

**Comment by Arunva28** [comment]
No, i am currently not accessing any network share. It's just from my HDD. When i select "Update Collection", there is a small bar coming up "Scanning collections" and disappears immediately.
